src/_derive/mod.rs

```diff
@@ -285,6 +285,8 @@
 //! - For custom type behavior, you can override the implied attributes/settings and/or set additional ones
 //! - `Option<Vec<T>>` will be `None` instead of `vec![]` if no arguments are provided.
 //!   - This gives the user some flexibility in designing their argument, like with `num_args(0..)`
+//! - To use `Vec<T>` with `short` or `long` options you need to specify `value_delimiter`, for
+//! example `value_delimiter = ','`
```

Comment:
This isn't correct.  [`Vec` implies `ArgAction::Append`](https://docs.rs/clap/latest/clap/_derive/index.html#arg-types) which means `-a b -a c` works.

Reply:
Thanks @epage I didn't realize that. What do you think of my updated version? I think it's worth mentioning because it wasn't obvious to me, so it might help other people too.

Reply:
imo I think the text is a bit confusing and in general I feel like this is scope creep for this piece of documentation.  We can't cover everyone's specific use case for a specialization that they want.

Reply:
Yeah, I think you're right, it's not great. But I've just noticed that there is a list of magic Arg attributes on that page and `value_delimiter` is not there, should I add it there instead? If not then feel free to close the PR.

Reply:
They keyword is "magic".  That is documented to mean that we do special processing for it.  `value_delimiter` falls under the category of "raw" attribute which is documented via the builder API.